### ISSUE TITLE
Show a different homepage depending on the value of a cookie or something

### ISSUE THREAD

**Issue by Ken-Lo** [gocardless/splash-pages#667]
Show a different homepage depending on the value of a cookie or something

@harrison @waltfy Any idea on the technical feasibility of doing this? I presume it's possible as we can use JS on splash pages, but not sure how this would work with the static site rendering?

**Background:** Currently re-designing the homepage to better funnel people towards the right section of the site. To help with activation, on-boarding & package up-sell, it would be good to show different content on the homepage for existing merchants (e.g. maybe a greater focus on how-to content, upgrading to Pro, etc).

**Comment by harrison** [comment]
@Ken-Lo yeah could do something similar to the `<Translation>` blocks, where we would embed all the versions in the js bundle, have the default in the static version and then swap it out with js. 
I'm a little bit hesitant to content like this as it makes testing and working on the site so much harder. We'll have to build some way of swapping your cookies on the dev version and possibly show that there are multiple versions so we don't forget when we do future changes to the main page.

**Comment by Ken-Lo** [comment]
Thanks @harrison - super useful! Would be really keen on having this as it would allow us to be so much more focussed with all of our messaging. Could we implement locally with different routes e.g. `/home-existing` for ease of testing locally, but dynamically switch the component when viewing live on the website?

**Comment by harrison** [comment]
Hm yeah not sure what would work best, ideally wouldn't require updating routes to add new content but should also be easy to view different versions. Would be good to scope this out properly when we decide to pick it up.

Let's track it here: https://gocardless.myjetbrains.com/youtrack/issue/UXR-34
We'll be prioritising from youtrack when deciding what requests to work on next.
